service_capacity_modeling/hardware/profiles/shapes/aws/auto_i7i.json

```diff
@@ -0,0 +1,220 @@
+{
+  "instances": {
+    "i7i.large": {
+      "name": "i7i.large",
+      "cpu": 2,
+      "cpu_cores": 1,
+      "cpu_ghz": 2.4,
```

Comment:
Oh so weird, so i7i has a sustained clock frequency lower than 3.2 the r7i they are based on are? I think a unit test that asserts the i7i has similar adjusted CPU perf as r7i might be a good one (and maybe paired over all i + r families)?

Reply:
Oh even weirder, their AWS [page](https://aws.amazon.com/ec2/instance-types/) says 3.2 (Powered by up to 3.2 GHz Intel Xeon Scalable Processors (Emerald Rapids 8559C) and vantage says that [too](https://instances.vantage.sh/aws/ec2/i7i.2xlarge)

Could the API just be wrong or is the base clock just not relevant anymore?